content/blog/excluding-targets-from-stack-operations/index.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Excluding targets from stack operations"
+date: 2025-04-30
+meta_desc: "You can now exclude targets from up, preview, refresh, and destroy operations"
+authors:
+    - tom-harding
+tags:
+    - exclude
+    - targets
+---
+
+Pulumi provides a set of top level commands for managing stack deployments. We can use commands like `up` and `destroy` to spin up and tear down production environments, or `refresh` to update our understanding of the stack’s resources.
```

Comment:
```suggestion
Pulumi provides a set of top-level commands for managing stack deployments. We can use commands like `up` and `destroy` to spin up and tear down production environments, or `refresh` to update our understanding of the stack’s resources.
```

Reply:
... I think it should be top-level, no? It's an adjective